docker-entrypoint.sh

```diff
@@ -27,8 +27,8 @@ file_env() {
     local var="$1"
     local fileVar="${var}_FILE"
     local def="${2:-}"
-    local varValue=$(env | grep -E "^${var}=" | sed -E -e "s/^${var}=//")
-    local fileVarValue=$(env | grep -E "^${fileVar}=" | sed -E -e "s/^${fileVar}=//")
+    local varValue="`env | grep -E \"^${var}=\" | sed -E -e \"s/^${var}=//\"`"
+    local fileVarValue="`env | grep -E \"^${fileVar}=\" | sed -E -e \"s/^${fileVar}=//\"`"
```

Comment:
why replace the newer `$()` syntax with the (what I believe) older and less readable \` syntax?

Reply:
@NeroBurner I just tried to replace this at first to see if it was causing the strange behavior. You can probably put them back. 
BTW I find backquotes way more readable :wink: